Runtime/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/RuntimeContentProvider.java

```diff
@@ -75,19 +75,20 @@ public boolean onCreate() {
    * @return Configuration for {@link TrackingManager}, may be null
    */
   @Nullable
-  private Config createConfig(@NonNull Context context, @Nullable ConfigurationResult lastConfig,
-      @Nullable String appId) {
-    PackageManager packageManager = context.getPackageManager();
-    String packageName = context.getPackageName();
+  private Config createConfig(@NonNull final Context context, @Nullable final ConfigurationResult lastConfig,
+      @Nullable final String appId) {
     if (lastConfig == null) {
       return null;
     }
+    PackageManager packageManager = context.getPackageManager();
+    String packageName = context.getPackageName();
```

Comment:
are the above 2 lines necessary? i don't see them being used in the new code. 

- [ ] remove unused code

Reply:
Oh, yes, packageName is used to get ApplicationInfo afterwards.
Those two lines are old code, I just have moved "if (lastConfig == null)" before them follow pmd report.

Reply:
ahh sorry i was only looking below and not above, my bad for misreading the diff